src/index.ts

```diff
@@ -1,18 +1,54 @@
 import joplin from 'api';
 import { MenuItemLocation } from 'api/types';
 import { DiffWindow } from './DiffWindow';
+import { NoteSelectWindow } from './NoteSelectWindow';
 
-let diffWindow;
+let diffWindow : DiffWindow;
+let noteSelectWindow : NoteSelectWindow;
 
 async function openDiffWindow(noteIds : string[]) {
 	if(noteIds.length !== 1) {
 		// Comparing multiple notes not yet supported.
 		return;
 	}
+
 	const noteId = noteIds[0];
+	const localNote = await joplin.data.get(['notes', noteId], {
+		fields: ['is_conflict', 'conflict_original_id']
+	});
+
+	if(localNote.is_conflict === 0) {
+		return await joplin.views.dialogs.showMessageBox("This is not a conflict note.");
+	}
+
+	let compareWithId = localNote.conflict_original_id;
+	if(compareWithId === "") {
+		compareWithId = await noteSelectWindow.openDialog();
+	}
+
+	if(compareWithId == "") {
```

Comment:
I see you alternate between equality and strict equality. If you have some time perhaps you could setup eslint so that it detects and fixes these inconsistencies.

Reply:
Oh good idea! I'll get it setup for sure cause I keep forgetting and doing double == by mistake haha.

Reply:
Done although it made me change a heck ton of stuff 😂

Reply:
It's a bit late for this, but in general it's better to make this kind of cosmetic changes in a separate pull request, otherwise it makes it hard to see what's actually been changed.

Reply:
> It's a bit late for this, but in general it's better to make this kind of cosmetic changes in a separate pull request, otherwise it makes it hard to see what's actually been changed.

Uhh true, ngl should have atleast focused on the stuff relating to this PR instead of linting everything haha. Myb.
Noted for next time.